hack/pull_kfp_upstream.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+export PIPELINES_SRC_REPO=https://github.com/kubeflow/pipelines.git
+export PIPELINES_VERSION=1.0.0-rc.1
```

Comment:
in the future for new version updates, we just 

1. update this value

2. run this shell?

Reply:
Yes, exactly